tests/UI/campaigns/functional/API/02_endpoints/09_product/04_patchProductId.ts

```diff
@@ -183,30 +193,210 @@ describe('API : PATCH /product/{productId}', async () => {
     });
 
     [
-      // @todo : https://github.com/PrestaShop/PrestaShop/issues/35626
+      {
+        propertyName: 'active',
+        propertyValue: false,
+      },
+      {
+        propertyName: 'additionalShippingCost',
+        propertyValue: faker.number.float({fractionDigits: 2}),
+      },
+      // @todo : https://github.com/PrestaShop/PrestaShop/issues/38787
```

Comment:
I believe that's because `availableDate` expects a date, not a datetime So the time part is always reduced/truncated to zero anyway So you'd need to test this with only a date without the `H:is` part

That said maybe the Swagger part should be able to give an example without the time part to prevent confusion